spec/soapy_bing/ads/bulk/parsers/csv_parser_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'json'
```

Comment:
Shouldn't we just put these `require`s in `lib/soapy_bing.rb`, so we don't have weird load errors later?